Dfe.Data.SearchPrototype/Web/Views/Home/Index.cshtml

```diff
@@ -19,54 +19,82 @@
             </button>
         </govuk-input-suffix>
     </govuk-input>
-</form>
-@* end of search component *@
-
-@* display results *@
+    @* end of search component *@
 
-@if (Model != null)
-{
-    @if (Model.HasResults)
+    @* filters *@
+    @if (Model != null)
     {
-        @* list results component to be inserted as partial view*@
-        <div class="govuk-grid-column-two-thirds">
-            <h3 class="govuk-heading-m">@Model.SearchResultsCount @(Model.HasMoreThanOneResult ? "Results" : "Result")</h3>
-            <hr />
-            <ul style="list-style: none">
-                @foreach (var searchItem in Model.SearchItems!)
+        <div class="govuk-grid-column-one-third">
+            <h3>Filters</h3>
+            @if (Model.Facets is not null && Model.Facets.Any())
+            {
+                @foreach (var facet in Model.Facets)
                 {
-                    <li>
-                        <h4>
-                            <a href="/">
-                                @searchItem.Name
-                            </a>
-                        </h4>
-                    </li>
-                    <li>
-                        <strong>URN:  </strong>@searchItem.Urn
-                    </li>
-                    <li>
-                        <strong>Address: </strong> @searchItem.AddressAsString()
-                    </li>
-                    <li>
-                        <strong>Type of establishment: </strong> @searchItem.EstablishmentType
-                    </li>
-                    <li>
-                        <strong>Status: </strong> @searchItem.EstablishmentStatusName
-                    </li>
-                    <li>
-                        <strong>Education phase: </strong> @searchItem.PhaseOfEducation
-                        <hr />
-                    </li>
-
+                    <govuk-checkboxes name="@facet.Name">
+                        <govuk-checkboxes-fieldset>
+                            <govuk-checkboxes-fieldset-legend is-page-heading="false" class="govuk-fieldset__legend--s">
+                                @facet.Name
+                            </govuk-checkboxes-fieldset-legend>
+                            @foreach (var facetItem in facet.Values)
+                            {
+                                <govuk-checkboxes-item value="@facetItem.Value">
```

Comment:
At some point, let's update this to be checked/not.

```
<govuk-checkboxes-item value="@facetItem.Value" checked="@facetItem.IsSelected">
```

Reply:
yes i'll add this to the board to implement on the other side?